demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -846,6 +847,7 @@ export class DemoMeetingApp
     const buttonCloudCapture = document.getElementById('button-record-cloud') as HTMLButtonElement;
     buttonCloudCapture.addEventListener('click', _e => {
       this.toggleButton('button-record-cloud');
+      this.updateButtonVideoRecordingDrop()
```

Comment:
I see you added a new method, but, there is an existing `toggleButton` just above for specific buttons should that be removed?

Reply:
Yes. This is to update states. Not simply to toggle.